salt/modules/transactional_update.py

```diff
@@ -941,10 +944,18 @@ def call(function, *args, **kwargs):
     activate_transaction = kwargs.pop("activate_transaction", False)
 
     try:
+        # Set default salt-call command
+        salt_call_cmd = "salt-call"
+        python_exec_dir = os.path.dirname(sys.executable)
+        if "venv-salt-minion" in pathlib.Path(python_exec_dir).parts:
+            # If the module is executed with the Salt Bundle,
+            # use salt-call from the Salt Bundle
+            salt_call_cmd = os.path.join(python_exec_dir, "salt-call")
```

Comment:
Why is it needed to do the  path juggling inside Python code, can't we prepend the "Salt Bundle" bin/ directory to `$PATH`?

Reply:
Yes, we can do it with PATH outside, but actually I think to do it this way is more raliable as it's more strict and in case of handling transactional_update with salt-ssh it will push us to make the external solution more conditional